pkg/collector/pipeline.go

```diff
@@ -102,15 +127,27 @@ func (p *Pipeline) ProcessReports(ctx context.Context, w http.ResponseWriter, r
 	err = decoder.Decode(&reports.Reports)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
-		return nil
+		return true
```

Comment:
Why does this case count as dropped while the cases above don't? Feels inconsistent to me.

Reply:
That is a mistake and should be false.